templates/textcat_prompt.jinja2

```diff
@@ -0,0 +1,38 @@
+{% if labels|length == 1 %}
+    {% set label = labels[0] %}
```

Comment:
> Is the indentation here needed? It seems to re-appear in the prompt.

Ah agree it's not needed and didn't intend it there. Just got used to the indentation within if else. Will fix this tomorrow